### ISSUE TITLE
Music doesn't play

### ISSUE THREAD

**Issue by StupidHoroscope** [StupidHoroscope/fks-eduke32#3]

**Comment by DrUm78** [comment]
There is no music because the OS does not include Timidity midi lib which is required by eDuke:
```Initializing music...
Error: couldn't open any of the following files:
/etc/timidity/freepats.cfg
/etc/timidity/timidity.cfg
/etc/timidity.cfg
Error: couldn't open any of the following files:
/etc/timidity/freepats.cfg
/etc/timidity/timidity.cfg
/etc/timidity.cfg
S_MusicStartup(): failed initializing
Initializing sound...
```
Gonna have a look whether I can add the package to my custom OS then include it in the OPK as well.

**Comment by StupidHoroscope** [comment]
Fixed in v1.1
